CHANGELOG/6.2.md

```diff
@@ -1,5 +1,20 @@
 # 6.2 Changelog
```

Comment:
@adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-windows`
Job Cmdlet Tests.Retrieving partial output from jobs.Receive-Job will retrieve partial output, including -Keep results
```powershell
Destination array was not long enough. Check the destination index, length, and the array's lower bounds. (Parameter 'destinationArray')
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Job.Tests.ps1: line 108
108:                         $results = Receive-Job -Keep $job

```